common/unsafe/src/main/java/org/apache/spark/sql/catalyst/util/CollationFactory.java

```diff
@@ -704,6 +789,52 @@ private String collationName() {
         }
         return builder.toString();
       }
+
+      @Override
+      protected Meta buildMeta() {
+        return new Meta(
+          COLLATION_CATALOG,
+          COLLATION_SCHEMA,
+          collationName(),
+          ICULocaleMap.get(locale).getLanguage(),
+          ICULocaleMap.get(locale).getCountry(),
```

Comment:
Could we possible change here to getDisplayCountry() and getDisplayLanguage(). The idea is to later be able to look for locales not only on name, but also on country name i.e. Greece, France...

Reply:
Sure